x-pack/plugins/security_solution/public/resolver/view/resolver_no_process_events.tsx

```diff
@@ -41,6 +41,14 @@ export const ResolverNoProcessEvents = () => (
       })}
     </EuiText>
     <EuiSpacer size="m" />
+    <EuiText size="s">
+      {i18n.translate('xpack.securitySolution.resolver.noProcessEvents.dataView', {
+        defaultMessage: `In case you selected a different data view,
+          make sure your data view contains one of the inidices that are stored in the source event at "{field}".`,
```

Comment:
instead of 1, should probably be "all" of the indices (spelling as well)

Reply:
👍 https://github.com/elastic/kibana/pull/122859/commits/80c18ed8993d7eeb805328b7e1c11b17db7e0f25